### DISCUSSION TITLE
建议R68S固件增加 挂载 SMB/CIFS 网络共享 这个功能（应该是个插件）

### DISCUSSION THREAD

**Discussion by tiny-andr** [coolsnowwolf/lede#9681] [Ideas / 建议]
刚需啊。目前只有一个共享出去的服务，却没有可挂载远端共享的服务（指用鼠标点的~）。
就是这个。
![image](https://user-images.githubusercontent.com/71067099/176931752-8e471c7f-9e5b-48e9-93cb-c0f02605df4a.png)

**Comment by xinpenghan** [comment]
很早就有了,make menuconfig里添加下面这两个.
 
CONFIG_PACKAGE_luci-app-cifs-mount=y
CONFIG_PACKAGE_luci-app-cifsd=y
